src/adapters/scene_direction.js

```diff
@@ -65,10 +65,12 @@ export default class SceneDirection {
   setMainRoute(routeId) {
     let mainRoute = null;
     this.routes.forEach((route) => {
-      if (route.id === routeId) {
+      const isActive = route.id === routeId
+      if (isActive) {
         mainRoute = route
       }
-      this.map.setFeatureState({source: `source_${route.id}`, id: 1}, {isActive: route.id === routeId})
+      this.map.setFeatureState({source: `source_${route.id}`, id: 1}, {isActive})
+      this.map.setLayoutProperty(`route_${route.id}`, 'icon-image', isActive ? 'walking_bullet_active' : 'walking_bullet_inactive')
```

Comment:
On a car or bicycle route (without `icon-image` layout) this will break the alternative route selection, with an error:
```
TypeError: this._values[t] is undefined
    getValue mapbox-gl.js:29
    getLayoutProperty mapbox-gl.js:29
    setLayoutProperty mapbox-gl.js:33
    setLayoutProperty mapbox-gl.js:33
    setMainRoute scene_direction.js:73
...
```